code/modules/language/language_holder.dm

```diff
@@ -17,6 +17,9 @@
 /datum/language_holder/Destroy()
 	owner = null
 	QDEL_NULL(language_menu)
+	languages.Cut()
+	showdow_languages.Cut()
+	return ..()
```

Comment:
references to other things rarely prevent a qdel, its only if other things have references to this,